source/user-manual/api/getting-started.rst

```diff
@@ -254,19 +270,23 @@ Here are some of the basic concepts related to making API requests and understan
-    |         +----------------------+-----------------------------------------------------------------------+
-    |         | total_failed_items   | Total number of failed items.                                         |
-    |         +----------------------+-----------------------------------------------------------------------+
-    |         | failed_items         | List containing each of the failed items in the request.              |
-    +---------+----------------------+-----------------------------------------------------------------------+
-    | message |                      | Result description.                                                   |
-    +---------+----------------------+-----------------------------------------------------------------------+
+    +---------+----------------------+----------------------------------------------------------------------------------------------------------------+
+    | Field   | Optional Sub-fields  | Description                                                                                                    |
+    +=========+======================+================================================================================================================+
+    | data    | affected_items       | List with each of the successfully affected items in the request.                                              |
+    |         +----------------------+----------------------------------------------------------------------------------------------------------------+
+    |         | total_affected_items | Total number of successfully affected items.                                                                   |
+    |         +----------------------+----------------------------------------------------------------------------------------------------------------+
+    |         | total_failed_items   | Total number of failed items.                                                                                  |
+    |         +----------------------+----------------------------------------------------------------------------------------------------------------+
+    |         | failed_items         | List containing each of the failed items in the request.                                                       |
+    +---------+----------------------+----------------------------------------------------------------------------------------------------------------+
+    | message |                      | Result description.                                                                                            |
+    +---------+----------------------+----------------------------------------------------------------------------------------------------------------+
+    | error   |                      | For HTTP ``200`` responses determines if the response was complete (``0``), partial (``2``) or failed (``1``). |
```

Comment:
Failed before partial to keep numbers order?

Reply:
fixed